t/pkg-Cpanel-Security-Advisor-Assessors-Imunify360.t

```diff
@@ -191,6 +154,30 @@ subtest 'When Imunify360 is installed and licensed' => sub {
     cmp_deeply( $advice->[0], superhashof($expected), "It should say that the server is protected" ) or diag explain $advice;
 };
 
+subtest 'When the custom URL is present' => sub {
+    plan tests => 1;
+
+    $imunify->redefine( get_custom_url       => sub { 'https://example.com' } );
+    $imunify->redefine( is_product_licensed  => sub { 0 } );
+    $imunify->redefine( is_product_installed => sub { 0 } );
+
+    my $advice = get_advice();
+
+    ok( $advice->[0]->{advice}->{suggestion} =~ qr{https://example.com}, "It should change the link href" );
```

Comment:
It would be better to use like() for this comparison so that you get proper diagnostic output on failure.

ok() is really only good for boolean comparisons. Anything else and it falls short in diagnostics.